test_conformance/relationals/test_shuffles.cpp

```diff
@@ -260,33 +263,48 @@ char * get_order_name( ExplicitType vecType, size_t inVecSize, size_t outVecSize
     return orderName;
 }
 
-void    print_hex_mem_dump( const unsigned char *inDataPtr, const unsigned char * inDataPtr2, const unsigned char *expected, const unsigned char *outDataPtr, size_t inVecSize, size_t outVecSize, size_t typeSize )
+void print_hex_mem_dump(const unsigned char *inDataPtr,
+                        const unsigned char *inDataPtr2,
+                        const unsigned char *expected,
+                        const unsigned char *outDataPtr, size_t inVecSize,
+                        size_t outVecSize, size_t typeSize)
 {
-    char error [4096] = "";
-    strcat(error, "      Source: ");
-    for( unsigned int j = 0; j < inVecSize * typeSize; j++ )
+    auto print_hex_byte = [](unsigned char v) {
```

Comment:
Wondering if `hex_byte_to_string` might be a better name for this?  It doesn't _actually_ print anything itself, aside from using `ostringstream` (which you could arguably consider printing), but that's an internal detail of the lambda.

Reply:
Good point, I'll update the name.